app/controllers/photos_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PhotosController < ApplicationController
   before_action :set_photo, only: %i[ show edit update destroy ]
+  before_action :ensure_current_user_is_owner, only: [:destroy, :update, :edit]
```

Comment:
Nice way of keeping user away from changing other users' photos and restricting available actions